app/views/Dashboard/index.tsx

```diff
@@ -161,14 +161,15 @@ function Dashboard(props: Props) {
                     calendarHeatmapHidden
                     handleDateRangeChange={setDateRangeSafe}
                     // eslint-disable-next-line max-len
-                    contributionTimeStats={filteredCommunityStats?.filteredStats?.contributorTimeStats}
+                    contributionTimeStats={filteredCommunityStats?.communityFilteredStats?.swipeTimeByDate}
+                    // eslint-disable-next-line max-len
+                    areaSwipedByProjectType={filteredCommunityStats?.communityFilteredStats?.areaSwipedByProjectType}
                     // eslint-disable-next-line max-len
-                    areaSwipedByProjectType={filteredCommunityStats?.filteredStats?.areaSwipedByProjectType}
+                    organizationTypeStats={filteredCommunityStats?.communityFilteredStats?.swipeByOrganizationName}
                     // eslint-disable-next-line max-len
-                    organizationTypeStats={filteredCommunityStats?.filteredStats?.organizationTypeStats}
-                    swipeByProjectType={filteredCommunityStats?.filteredStats?.swipeByProjectType}
+                    swipeByProjectType={filteredCommunityStats?.communityFilteredStats?.swipeByProjectType}
                     // eslint-disable-next-line max-len
-                    contributions={filteredCommunityStats?.filteredStats?.projectGeoContribution as MapContributionType[] | undefined}
+                    contributions={filteredCommunityStats?.communityFilteredStats?.swipeByProjectGeo as MapContributionType[] | undefined}
```

Comment:
Double check if this cast is required.

Reply:
It is required, yes. The type of `geoJSON` that comes from graphql has a field with type 'unknown' and is incompatible with what we need.